src/components/annual-report/report-data/section4/section4Calculations.js

```diff
@@ -0,0 +1,158 @@
+import { getRoundedValue } from '../utils/mathService.js'
+import { mapSectionFourResponse } from '../utils/utils.js'
+
+const OVERALL_RESPONSE_NAME = 'data_over_all_org_units'
+const BY_LEVEL_RESPONSE_NAME = 'data_detail_by_level'
+const DENOMINATOR_RELATIONS_MAP = 'denominatorRelationsMap'
+
+const getVal = ({ response, dx, ou, pe }) => {
+    return response?.[dx]?.[ou]?.[pe]
+}
+
+const calculateSection4a = ({
+    formattedResponseOverall,
+    denominatorRelations,
+    currentPeriod,
+    overallOrgUnit,
+}) => {
+    const section4a = []
+    for (const rel of denominatorRelations) {
+        if (rel.type !== 'un') {
+            break
+        }
+        section4a.push({
+            name: rel.name,
```

Comment:
Does this particular section get a hard-coded name? I see "Ratio of population projection from the Bureau of Statistics to a UN projection" in the old app, but maybe that's a different object

Reply:
Yeah...this is kind of weird in the old app, but it let you define multiple denominator relations of type `un` and then only used the last one in the report and put it with that hardcoded name.

I spoke with Olav, and we decided to let you add multiple ones in the configuration, and have them all displayed in the report (hence we cannot use the hard-coded name).

(See slack conversation: https://dhis2.slack.com/archives/C053JT69YD8/p1696500142931519)